### DISCUSSION TITLE
[预览] v2.8.0

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#4234] [分享 / Share]
包含 v2.7.2 的所有更新内容.

✨新增

- `清爽首页` 改进:
  - 支持自定义板块的布局和可见性. (操作方式和自定义顶栏那个差不多)
  - 番剧时间表优化样式, 增加翻页按钮.

<div>
  <img height="400" src="https://user-images.githubusercontent.com/26504152/242905984-895cb72c-b344-40c3-91a0-2a6b20d5f783.png">
  <img height="400" src="https://user-images.githubusercontent.com/26504152/242905640-cbc948f1-734e-46f2-96a7-d57787b7cf47.png">
</div>

- 新增功能 `首页净化`. (PR #4153 by [RieN](https://github.com/rien7))
> 删除首页特定类型的卡片.

- `自定义顶栏` 增加 `创作中心` 入口, `排行` 新增 `全站音乐榜` 和 `短剧榜` 的入口. (#4101, PR #4154 by [星海](https://github.com/lovelyCARDINAL), PR #4155 by [星海](https://github.com/lovelyCARDINAL))
- `播放器置顶（新）` 增加选项 `顶部留白`. (#4152, PR #4165 by [Ziu](https://github.com/ZiuChen))
- `播放器置顶（新）` 的标题改为放置到播放器和点赞中间. (PR #4208 by [LockRim](https://github.com/LockRim))
- `显示视频投稿时间` 支持转义字符. (#4160, PR #4167 by [呼乎户](https://github.com/wisokey))
- `隐藏视频推荐` 支持稍后再看和收藏夹播放页, 并适配 bpx 播放器. (#4174)
- 新增功能 `隐藏记笔记` 和 `隐藏稿件投诉`. (#4124)
- 优化了 `自定义顶栏` 中订阅在筛选时的 API 调用. (#4120)
- 优化粗体的显示: 优先调用 Semibold 字重, 若字体不支持则使用 Bold 字重.
- `自定义顶栏` 增加选项:
  - 支持自定义消息提醒样式, 可设置为 `数字`, `点状` 或 `隐藏`. (#4125)
  - 支持自定义搜索栏宽度. (#4112)
- 搜索在纯数字输入时, 增加对 UID 和直播间的搜索. (#677)

☕开发者相关
- 播放器模式转换出的 bpx class 更换为更统一的名称, 并增加事件派发. (PR #4191 by [Ziu](https://github.com/ZiuChen))
- 需要选择组件 / 插件的构建任务中, 优化了选项的长度. (PR #4192 by [Ziu](https://github.com/ZiuChen))
- 弃用 utils 中的 `formData`. (名称具有歧义, 应使用 `URLSearchParams` 来代替)
- `addComponentListener` / `removeComponentListener` 支持传入多层的组件 options 路径.
- 移除依赖 `vue-fragment`. (使用 `display: contents` 代替)

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.8.0-preview'>[预览] v2.8.0</a>.</em>

**Comment by the1812** [comment]
有个组件开启 / 关闭时不能立即生效的 bug 修复, 需要将版本从 `v2.8.0-preview-72-ga581499e5` 更新至 `v2.8.0-preview-77-g7e99bde31`

**Comment by imshixin** [comment]
更新说明溢出啦
![image](https://github.com/the1812/Bilibili-Evolved/assets/36874714/40eea475-ff59-4c8e-aede-2e643e0e9de2)

**Reply by the1812** [reply]
这个也是本次更新的一个修复项

**Comment by u2shana** [comment]
新版本的直播侧边栏信息扩充的字体感觉有点小，是故意这样设定来容纳更长标题的么？

**Reply by the1812** [reply]
遵循旧版动态页的标准字体尺寸 (14px)

**Comment by Geeyun-JY3** [comment]
`首页净化`是干啥用的，试了一下没看出区别。
另外这针对的是新版首页（分区界限分明且非无限滚动）还是内测首页（无限滚动）。

***

2023-10-15 编辑：好的是新版首页（我是瞎子）：https://github.com/the1812/Bilibili-Evolved/pull/4153#issuecomment-1529044488

**Comment by pinaZC** [comment]
清爽首页的更新太好了！！！好评如潮！！！

**Comment by Tyi-13** [comment]
首页顶部插件影响白名单什么的那个警告黄条可以干掉吗？

**Reply by Geeyun-JY3** [reply]
这个横幅的出现和 Bilibili Evolved 就没关系：#3719、#3989

**Comment by lusa214** [comment]
![image](https://github.com/the1812/Bilibili-Evolved/assets/55358634/c01a2ba5-f6ae-4f07-8247-18fe1192d4ff)
请问为什么我装上去是2.5.2 QAQ

**Reply by lusa214** [reply]
![image](https://github.com/the1812/Bilibili-Evolved/assets/55358634/8cf45fe1-aa1c-4663-8c48-a9a4650ff847)
以及为什么快速收藏和稍后再看也没了 QAQ 求教

**Reply by Geeyun-JY3** [reply]
你这是什么安装方法？下的 release 附带的压缩包？

**Reply by lusa214** [reply]
对QAQ

**Reply by Geeyun-JY3** [reply]
谁教你这样安装的？安装方法认真阅读 README：https://github.com/the1812/Bilibili-Evolved#%E5%AE%89%E8%A3%85
怎么老有人这样安装，真是服了。

***

@the1812 建议把提醒写进 README 和在 release note 置顶。

**Reply by lusa214** [reply]
我知道问题在哪了，我用的不是预览版，所以是2.7.2= =
息怒息怒哈，已捐赠，买杯奶茶喝！

**Reply by Geeyun-JY3** [reply]
不管哪个版本都别再下 release 的压缩包了。。。
***
我不是开发者，捐赠也不是给我。
只是很好奇怎么老有人这样安装，几个月前就有这一批人也是这样下 release 的压缩包安装，之前也在 release note 挂过一段时间提醒。
不会是什么 GitHub 使用教程视频这样教的吧？
